com.b2international.snomed.ecl.ui/src/com/b2international/snomed/ecl/ui/EclUiModule.java

```diff
@@ -53,4 +57,11 @@ public EclUiModule(AbstractUIPlugin plugin) {
 		return EclUiValidator.class;
 	}
 	
+	public Class<? extends AbstractAntlrTokenToAttributeIdMapper> bindAbstractAntlrTokenToAttributeIdMapper() {
+		return EclAntlrTokenToAttributeIdMapper.class ;
```

Comment:
Add these to the IDE module, please.

Reply:
Should I move every function from the UI module to the IDE module?

Reply:
No, keep them there, @apeteri will move them if needed for the lang server development.

Reply:
The ide module was removed. @cmark: ignore